design/ceph/object/store.md

```diff
@@ -712,3 +712,73 @@ For external CephCephObjectStores (i.e., when `spec.gateway.externalRgwEndpoints
 vhost-style addressing should be configured on the host cluster, and `hosting.dnsNames` is
 irrelevant. The default `advertiseEndpoint` for external CephObjectStores is the first entry in the
 `spec.gateway.externalRgwEndpoints` list, which users should be able to override if desired.
+
+### RGW Operations Logging
+
+If you want to have the ops log available for enhanced observality of the operations, you'd have to set `rgwOpsLogEnabled` this will set the [configuration for enabling ops logs to Ceph](https://docs.ceph.com/en/latest/radosgw/config-ref/#confval-rgw_enable_ops_log).
+
+```yaml
+spec:
+  metadataPool:
+    replicated:
+      size: 1
+  dataPool:
+    replicated:
+      size: 1
+  preservePoolsOnDelete: false
+  gateway:
+    port: 80
+    # securePort: 443
+    instances: 1
+  rgwOpsLogEnabled: true
```

Comment:
There are other logging settings that could potentially be exposed. How about if we have a `logging` element, with the settings under it like so:
```suggestion
  gateway:
    logging:
      opsLogEnabled: true
```
Since it's part of the rgw pod, it should also be under the `gateway` settings.

An alternative to consider if there were other ops log settings:
```yaml
  gateway:
    logging:
      opsLog:
         enabled: true
```